packages/account/src/providers/transaction-request/transaction-request.ts

```diff
@@ -457,36 +449,6 @@ export abstract class BaseTransactionRequest implements BaseTransactionRequestLi
     return this;
   }
 
-  /**
-   * Adds multiple resources to the transaction by adding coin/message inputs and change
-   * outputs from the related assetIds.
-   *
-   * @param resources - The resources to add.
-   * @returns This transaction.
-   */
-  addPredicateResource(resource: Resource, predicate: Predicate<InputValue[]>) {
```

Comment:
Why is this being removed here? It's being removed in #2037 as well.

Reply:
@nedsalk Yeah I've removed it first here.

It would be good if we could merge https://github.com/FuelLabs/fuels-ts/pull/2037 before this one.

I'm just waiting for one more review since it is a tricky change